### PR TITLE
(TDP-4606): remove mixpanel and improve pendo

### PR DESCRIPTION
**Pull request by ncomont**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-4606

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [x] The new code does not introduce new technical issues (sonar / eslint)
- [x] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [x] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/962/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/962/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/965/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/965/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/979/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/979/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/981/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/981/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/989/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/989/)

File: dataprep-webapp/src/app/settings/settings-service.spec.js [hunk 1]
```diff
@@ -45,28 +45,32 @@ describe('Settings service', () => {
...
 
 			// when
 			SettingsService.refreshSettings();
 			$httpBackend.flush();
 
 			// then
-			expect(appSettings).toEqual(settings);
+			expect(appSettings).toEqual({
+				...settings,
+				analytics: {},
```
Comment (jsomsanith) [inline dataprep-webapp/src/app/settings/settings-service.spec.js]: Why don't you add analytics mocks instead ?

**Pull request merged**